keras_core/distribute/distribution.py

```diff
@@ -0,0 +1,144 @@
+"""Unified high level distribution APIs across backends.
```

Comment:
Please rename to `keras_core/distribution/distribution_lib.py`

Reply:
Done.